.github/dependabot.yml

```diff
@@ -0,0 +1,53 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/frontend"
+    schedule:
+      interval: "daily"
```

Comment:
Could we change this intervals, to monthly or weekly. I am not a huge fan of the noice generated by dependabot especially on nodejs applications ?

Reply:
i set daily instead of weekly to have small incremintal updates, to be honnest i rather prefer have 1 dependabot notification in 2 days than 5 at once on sunday !
But hey, i have no problem setting up that on weekly too  🤷🏾‍♂️ !

Reply:
Well let's go with the daily and see how annoying it gets. We can always change that to something else 👌